zebrad/src/application.rs

```diff
@@ -342,7 +342,11 @@ impl Application for ZebradApp {
             .as_ref()
             .expect("config is loaded before register_components");
 
-        let default_filter = if command.verbose { "debug" } else { "info" };
+        let default_filter = if command.verbose {
+            "debug,rpc=trace"
+        } else {
+            "info,rpc=trace"
+        };
```

Comment:
Performance/Usability: lightwalletd makes 1.6 million RPC requests over 30 minutes when doing its initial block ingestion run.

So here's what I'd suggest instead:
1. Keep `info` as the default level in production
2. Log the first RPC call for each method at `info`-level, and the rest at `trace` (using a `Metadata` containing a list of called methods)
3. Show `info`-level `zebra_rpc` logs in the tests

Here are the test log filters:
https://github.com/ZcashFoundation/zebra/blob/5e1fd4b2d6dd663b0c67968c20f8e420f14d9d9b/zebra-test/src/lib.rs#L73-L76

Reply:
That seems like a lot of complexity for something that's mostly temporary, to be used during development.

It might be easier to just keep the previous default and update the test execution to use a custom Zebra config.

Reply:
I think you're right, we can achieve our testing goals by using different tracing filters in each test.

Reply:
I reverted the change. We can add the `[tracing] filter = "debug,rpc=trace"` configuration to `zebrad.toml` files for specific tests that need it.